src/DotNetty.Transport/Channels/ChannelInboundHandler.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DotNetty.Transport.Channels
+{
+    public interface ChannelInboundHandler : IChannelHandler
```

Comment:
>ChannelInboundHandler [](start = 21, length = 21)

there's no real need in ChannelInboundHandler analog in DotNetty. DotNetty port was originally based on netty 5.0 that has since been canceled. We've since aligned it better with netty 4.1 but kept handling of inbound vs outbound based on more granular/flexible model available in 5.0. In that model there's no real need in explicit inbound vs outbound declaration through implementation of inbound vs outbound interface. Methods that you override when inheriting from ChannelHandlerAdapter define whether the handler is inbound, outbound or both. Hence there's no need in ChannelInboundHandler, ChannelInboundHandlerAdapter. It's OK to just inherit SimpleChannelInboundHandler from ChannelHandlerAdapter directly.